packages/flutter/lib/src/services/raw_keyboard_web.dart

```diff
@@ -38,6 +40,12 @@ class RawKeyEventDataWeb extends RawKeyEventData {
   /// for more information.
   final String key;
 
+  /// The `KeyboardEvent.location` corresponding to this event.
```

Comment:
Maybe explain a bit more in detail? what is the KeyboardEvent.location? or give a example

Reply:
I added some explanation to each field, but generally they're fields of Web's `KeyboardEvent` class, as linked at the end of each comment.